FindPeakElement.java

```diff
@@ -9,8 +9,9 @@
  * 
  * Space Complexity: O(1) Time Complexity: O(log n)
  * 
- * Not yet submitted, the following code contains bugs
- * TODO: Fix the Bug
+ * Leetcode Result:
```

Comment:
mention this as well!!

Reply:
Do you want me to add a description about what I fixed?